webapp/src/components/mobilitymap/InstructionsModal.js

```diff
@@ -0,0 +1,259 @@
+import api from "../../api";
+import useToggle from "../../state/useToggle";
+import React from "react";
+import Accordion from "react-bootstrap/Accordion";
+import Alert from "react-bootstrap/Alert";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+
+const InstructionsModal = () => {
+  const showModal = useToggle(false);
+  const [accordionKey, setAccordionKey] = React.useState("");
+  const [version, setVersion] = React.useState("");
+  const toggleShow = () => showModal.toggle();
+  React.useEffect(() => {
+    if (!accordionKey) {
+      api.getUserAgent().then((r) => {
+        if (r.data) {
+          const browser = r.data;
+          if (browser.device === "Uknown") {
+            return;
+          }
+          if (browser.isIos) {
+            setAccordionKey("ios");
+            return;
+          }
+          if (browser.isAndroid) {
+            setVersion(browser.platformVersion);
+            setAccordionKey("android");
+            return;
+          }
+          setAccordionKey(browser.device.toLowerCase());
+        }
+      });
+    }
+  }, [accordionKey]);
+
+  return (
+    <>
+      <Button
+        variant="darksuma"
+        size="sm"
+        className="w-100 mt-2 text-white"
+        onClick={toggleShow}
+      >
+        <i className="bi bi-book"></i> Location Instructions
+      </Button>
+      <Modal show={showModal.isOn} onHide={toggleShow}>
+        <Modal.Header closeButton>
+          <Modal.Title>Enable Location Services</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Alert variant="info">
+            <p>
+              <i className="bi bi-info-circle-fill"></i> Click on your browser/device
+              below and follow the instructions to enable location service.
+            </p>
+            <p className="m-0">
+              <b>Refresh</b> this page after you are done following instructions.
```

Comment:
`Can you do use a link/button that does `window.location.refresh()` on click, to reload the page?